### PR TITLE
Thank supporters of Q-Phase and Queerbeats

### PR DESCRIPTION
**Pull request by thsmdt**
Hi there 🥳,

I just thought it might be a nice gesture to thank supporters of our events from last weekend on the website for a little while.

Further, I would recommend not removing both pages right away, as there may still be posters and flyers on display somewhere. From my perspective, we should serve those links for a little longer, at least.

### CODE REVIEW

File: src/pages/q-phase-21.tsx [hunk 1]
```diff
@@ -86,22 +35,17 @@ export default function QPhase21(): ReactElement {
...
-            (Perso, Reisepass, etc.) mit dabei haben. Das Hygienekonzept zur
-            Q-Phase kannst du dir{" "}
-            <Link
-              href="https://cloud.queerbeet.org/index.php/s/S7NLkGSF27dtYGd"
-              target="_blank"
-            >
-              hier
+            <b>Und schon ist es wieder rum!</b>
+            <br />
+            Wir hoffen, dass Ihr eine die Zeit genauso genossen habt wie wir.
```
Comment (erikgoe) [inline src/pages/q-phase-21.tsx:40]:
```suggestion
            Wir hoffen, dass Ihr die Zeit genauso genossen habt wie wir.
```

File: src/pages/q-phase-21.tsx [hunk 2]
```diff
@@ -86,22 +35,17 @@ export default function QPhase21(): ReactElement {
...
+            Hoffentlich sehen wir uns bald bei unseren regelmäßigen
+            Kaffeeklatsch wieder 🥳. Danke auch an unsere fleißigen Helfer, die
```
Comment (erikgoe) [inline src/pages/q-phase-21.tsx:42]: "Kaffeeklatschen" plural or "unserem regelmäßigen Kaffeeklatsch"?

File: src/pages/queerbeats.tsx [hunk 3]
```diff
@@ -71,14 +66,17 @@ export default function Queerbeats(): ReactElement {
...
-            queerbeats nun auf eine 2G+ Regelung setzen. Dies bedeutet, dass
-            beim Einlass neben dem 2G-Nachweis (vollständig geimpft oder
-            genesen) nun zusätzlich ein negativer Schnelltest (nicht älter als 24h)
-            oder negativer PCR-Test (nicht älter als 48h) vorzuzeigen ist.
+            <b>Danke für eure Teilnahme!</b>
+            <br />
+            Leider ist es schon wieder rum. Weder die Maskenpflicht noch die
+            kühlen Temperaturen konnten uns auf dem Dancefloor etwas anhaben.
+            Falls ihr den Abend genauso gelungen fandet, sehen wir euch
+            vielleicht bald auch bei unseren Kaffeeklatsch wieder. Danke auch an
```
Comment (erikgoe) [inline src/pages/queerbeats.tsx:77]: "Kaffeeklatschen" plural or "unserem Kaffeeklatsch" (same as above)?

File: src/pages/queerbeats.tsx [hunk 4]
```diff
@@ -71,14 +66,17 @@ export default function Queerbeats(): ReactElement {
...
+            unsere ganzen Helfer und das{" "}
```
Comment (erikgoe) [inline src/pages/queerbeats.tsx:78]: "unsere ganzen Helfer" sounds funny :smile: Maybe something like "unsere tollen Helfer" or just "unsere Helfer"?

Comment (lorenzo-w) [reply src/pages/queerbeats.tsx:78]: Sollte auch Helfer*innen heißen 😇

File: src/pages/q-phase-21.tsx [hunk 5]
```diff
@@ -86,22 +35,17 @@ export default function QPhase21(): ReactElement {
...
-            <Link
-              href="https://cloud.queerbeet.org/index.php/s/S7NLkGSF27dtYGd"
-              target="_blank"
-            >
-              hier
+            <b>Und schon ist es wieder rum!</b>
+            <br />
+            Wir hoffen, dass Ihr die Zeit genauso genossen habt wie wir.
+            Hoffentlich sehen wir uns bald bei unseren regelmäßigen
+            Kaffeeklatsch wieder 🥳. Danke auch an unsere fleißigen Helfer, die
```
Comment (lorenzo-w) [inline src/pages/q-phase-21.tsx:42]:
```suggestion
            Kaffeeklatschen wieder 🥳. Danke auch an unsere fleißigen Helfer, die
```

Comment (lorenzo-w) [inline src/pages/q-phase-21.tsx:42]:
```suggestion
            Kaffeeklatsch wieder 🥳. Danke auch an unsere fleißigen Helfer*innen, die
```

File: src/pages/queerbeats.tsx (hunk 3 above)
Comment (lorenzo-w) [inline src/pages/queerbeats.tsx:77]:
```suggestion
            vielleicht bald auch bei unseren Kaffeeklatschen wieder. Danke auch an
```

File: src/pages/queerbeats.tsx (hunk 4 above)
Comment (lorenzo-w) [inline src/pages/queerbeats.tsx:78]:
```suggestion
            unsere Helfer*innen und das{" "}
```

**Review by lorenzo-w — COMMENTED** [review]
Some minor grammar issues, otherwise a really nice and well-written gesture :)

**Comment by thsmdt** [comment]
@erikgoe @lorenzo-w Thank you for your reviews! 🤗

I applied the changes in a single commit and reformatted the files.

**Review by lorenzo-w — APPROVED** [review]

**Pull request merged**